src/main/java/ca/gc/cyber/ops/assemblyline/java/client/clients/AssemblylineClient.java

```diff
@@ -413,12 +414,17 @@ protected void setSession(ClientResponse clientResponse) {
         return Mono.just(rc);
     }
 
-    private String extractApiErrorMessage(WebClientResponseException exception) throws JsonProcessingException {
+    private String extractApiErrorMessage(WebClientResponseException exception) {
+        var exceptionMessage = exception.getResponseBodyAsString();
```

Comment:
Can this be put directly in the `return` statement so that it only gets executed when it's needed?